soh/soh/Enhancements/randomizer/3drando/entrance.hpp

```diff
@@ -180,6 +183,14 @@ class Entrance {
         return primary;
     }
 
+    bool IsDecoupled() const {
+        return decoupled;
+    }
+
+    void SetDecoupled() {
+        decoupled = true;
+    }
```

Comment:
i'm assuming we never want to unset decoupled so this should be fine as is, but part of me thinks it should be
```suggestion
    void SetDecoupled(bool decouple = true) {
        decoupled = decouple;
    }
```
just in case or something

Reply:
Yeah, I don't see that use case coming up (as depending when you'd want to change it, you'd have to reshuffle the entrances).